test/mux.sh

```diff
@@ -237,15 +250,25 @@ write_and_verify_logs() {
 }
 
 reset_ES_HOST() {
-    oc set env dc logging-mux $1 $2
+    muxpod=$( get_running_pod mux )
+    os::cmd::try_until_success "oc set env dc logging-mux $1 $2"
     os::cmd::try_until_failure "oc get pod $muxpod"
+    os::log::debug $( oc get pods -l component=mux )
     os::cmd::try_until_text "oc get pods -l component=mux" "^logging-mux-.* Running "
```

Comment:
After this line returns, then the old mux has been removed and the new one is running.  Which means mux has been redeployed.  Which means you do not need to check for rollout status, and you do not need to check again that mux is running below.